setup.py

```diff
@@ -14,7 +14,7 @@
         "backoff>=1.8.0, <2",
         "python-dateutil==2.8.0",
         "ratelimit==2.2.1",
-        "facebook_business>=16.0.0",
+        "facebook_business>=18.0.0",
```

Comment:
Apparently, yesterday, Facebook released [v19.0.0](https://github.com/facebook/facebook-python-business-sdk/releases/tag/19.0.0) of the sdk (and in turn, the API). 

If it isn't too much work I think we should get that one instead of v18. After all, it will increase the time until we have to check this again. Less noice 🔕

Reply:
This one might be handy to check breaking changes: https://developers.facebook.com/blog/post/2024/01/23/introducing-facebook-graph-and-marketing-api-v19/